snap/pack/pack.go

```diff
@@ -122,7 +122,7 @@ func loadAndValidate(sourceDir string, yaml []byte) (*snap.Info, error) {
 		return nil, fmt.Errorf("cannot validate snap %q: %v", info.InstanceName(), err)
 	}
 
-	if err := snap.ValidateSnapContainer(container, info, logger.Noticef); err != nil {
+	if err := snap.ValidateSnapContainer(container, info, logger.Debugf); err != nil {
 		return nil, err
```

Comment:
Wondering if it makes sense to give context that we are busy with "container validation", similar to what is done for "snap validation" with "cannot validate snap %q:"...

but perhaps not worth the repercussions as part of short term minimal fix...

Reply:
I agree, it's better to avoid breaking scripts that depend on some regex until we have a good plan.